cardano_node_tests/tests/tests_conway/test_committee.py

```diff
@@ -341,6 +342,28 @@ def test_update_committee_action(
 
         reqc.cip007.success()
 
+        if configuration.HAS_DBSYNC:
+            # Check dbsync
+            dbsync_committee_threshold = list(dbsync_queries.query_new_committee_info(txhash=txid))
+            assert (
+                dbsync_committee_threshold[0].quorum_denominator == 3
+            ), "Incorrect committee threshold denominator in dbsync"
+            assert (
+                dbsync_committee_threshold[0].quorum_numerator == 2
+            ), "Incorrect committee threshold numerator in dbsync"
+
+            dbsync_committee_members = list(
+                dbsync_queries.query_committee_members(
+                    committee_id=dbsync_committee_threshold[0].id
+                )
+            )
+            size_of_proposed_cm = len(prop["proposalProcedure"]["govAction"]["contents"][2]) + len(
+                gov_state["committee"]["members"]
+            )
+            assert (
+                len(dbsync_committee_members) == size_of_proposed_cm
+            ), "The number of committee members doesn't match in dbsync"
+
```

Comment:
Can the be turned into general helper function in `dbsync_utils.py` that would just accept gov state dump and proposal id?